Components/FlightControl/Inc/WatchdogTask.hpp

```diff
@@ -0,0 +1,52 @@
+/**
+ ******************************************************************************
+ * File Name          : FlightTask.hpp
+ * Description        : Primary flight task, default task for the system.
+ ******************************************************************************
+*/
+#ifndef SOAR_WATCHDOGTASK_HPP_
+#define SOAR_WATCHDOGTASK_HPP_
+#include "Task.hpp"
+#include "SystemDefines.hpp"
+#include "RocketSM.hpp"
+#include "Timer.hpp"
+    
+/* Macros/Enums ------------------------------------------------------------*/
+enum RADIOHB_COMMANDS {
```

Comment:
Make this a general HEARTBEAT_COMMANDS enum

Reply:
done!